### ISSUE TITLE
Sex Chromosome

### ISSUE THREAD

**Issue by ValenzuelaLab** [Tree-of-Sex/toso#90]
### In which category is your new term request?

- [X] new_definition
- [ ] existing_definition

### Preferred term label

Sex Chromosome

### Synonyms

Allosome

### Synonym type

None

### Textual definition

Sex chromosomes are linkage groups whose inheritance is correlated with sex

### Reference

Bachtrog, D., M. Kirkpatrick, J. E. Mank, S. F. McDaniel, J. C. Pires, W. Rice, and N. Valenzuela. 2011. Are all sex chromosomes created equal? Trends Genet. 27:350-357.

### ORCID Identifier

0000-0003-1148-631X

### Additional context

Definitions in OLS are so broad (i.e. "A chromosome involved in sex determination") that all ESD taxa would have sex chromosomes as chromosomes carrying genes in ESD that sense or transduce the environmental signal during sex determination (to commit the bipotential gonad/individual to its sexual fate) could be consider sex chromosomes by this definition.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by KamilSJaron** [comment]
I quite like this definition 

- it covers well haplodiploidy
- covers well homomorphic sex chromosomes

What I really like is that it removes the direct link with sex determination.

I guess only strange situation is created for PGE species (fungus gnats, gall midges and globular springtails) - the inheritance of what we call sex chromosomes and autosomes is the same; sex chromosomes are the chromosomes that are eliminated (paternal copy) in early development of males (while paternal autosomes are eliminated during spermatogenesis). So their "inheritance" does not correlate with sex, it's quite specifically just sex specific karyotype due to elimination patterns. I wonder if there is some easy fix I don't see.

But we could also just give up on having a definition that works for the words we used for flies and springtails; Maybe we should just stop calling them sex chromosomes (although there are tons and tons of papers doing that, especially in flies they exist since 20s);

**Comment by ChiBenv** [comment]
Maybe a bit pedantic, but should we not define sex chromosomes as chromosomes containing linkage groups whose inheritance is correlated with sex?

**Comment by anyatorgasheva** [comment]
Would the issue with PGE be (at least partially) fixed if we add: present in all cells of the organism? Or should we define another term like "basic chromosome set" where we define that it excludes supernumarary chromosomes etc and refer to it in the definitions of autosomes and sex chromosomes?

**Comment by CEMcDonoughGoldstein** [comment]
What about making it even more general:

Sex chromosomes are chromosomes containing linkage groups that are distinguishable between sexes in a population

and could add a sentence with more specifics:
Inheritance of these chromosomes is often associated with sex and initiation of sexual development pathways may correspond to presence or copy number of these chromosomes.

**Comment by KamilSJaron** [comment]
I wonder if the field would have easy time accepting this definition, but I do think @CEMcDonoughGoldstein 's is covering the best everything we would call sex chromosomes!! We might need to tweak the additional info to talk about relative copy number if we don't want to say that all chromosomes in haplodiploids are sex chromosomes, but that's the only obscurity I could think of.

**Comment by CEMcDonoughGoldstein** [comment]
I think a strong argument in support might be that this definition matches how sex chromosomes were originally searched for and identified (and we can have Nettie Steven's papers as one of the references). And that history has defined and shaped our understanding.

A sentence for the haplodiploid caveat could be:

If all chromosomes differ between sexes (such as in haplodiploidy) then typically none are classified as sex chromosomes.

**Comment by KamilSJaron** [comment]
And we could add a note that not all sex chromosomes contain genes that determine sex, nor all genes involved in sex determination are located on sex chromosomes.

**Comment by KamilSJaron** [comment]
Sex chromosome formula states from ToS1:

X0 #40 
complex XY #45 
ZW
complex X0
complex ZW #48
homomorphic

Those we want need a definition!

Do we want to keep homomorphic? Should we also capture details about the complex systems in a different column? (i.e. suing a different key in the ontology)

**Comment by ASEMBER2** [comment]
I do not agree that "Sex chromosomes are chromosomes containing linkage groups" because sex chromosomes ARE linkage groups. 

Also "allosome" is meant only for heterogametic chromosome 
 Y or W.

Inheritance correlated with sex is also in GRC e.g. in birds or in B chromosomes (even if they do not play direct role in sex determination) (this was already noted above).

My take (considering all the above comments) is:

Sex chromosomes are linkage groups whose inheritance is usually associated with sex and initiation of sexual development pathways, achieved by the presence or copy number of these chromosomes.

Not all sex chromosomes contain genes that determine sex, nor all genes involved in sex determination are located on sex chromosomes. If all chromosomes differ between sexes (such as in haplodiploidy) then typically none are classified as sex chromosomes.

**Comment by CEMcDonoughGoldstein** [comment]
I think that we can avoid the issue with chromosome/linkage group by just saying it is a chromosome and then chromosome will be defined elsewhere.

I disagree that we should have the definition of sex focus on sex determination - the exception is so broad. I think we should have all of that in the additional information.

My suggestion:

Sex chromosomes are chromosomes that are distinguishable between sexes in a population.

Inheritance of sex chromosomes is usually associated with sexual phenotype and initiation of sexual development pathways, achieved by the presence or copy number of these chromosomes. Not all sex chromosomes contain genes that determine sex, nor all genes involved in sex determination are located on sex chromosomes. If all chromosomes differ between sexes (such as in haplodiploidy) then typically none are classified as sex chromosomes.

**Comment by ccaio** [comment]
Ontology meeting discussion @ccaio and @ValenzuelaLab:

Based on the operational definition (what we need to define for input of X,W,Y etc. in the database) we thought it would be more appropriate to select most of Alex's definition and . 

**Definition:**
Sex chromosomes are linkage groups whose inheritance is associated with sex and the initiation of sexual development

**Parent:**
chromosome

**Children:**

- X chromosome #11 
- Y chromosome 
- W 
- Z #20 
- U
- V
- etc.

**Comment by ValenzuelaLab** [comment]
Reading again earlier comments about haplodiploidy and PGE I noticed that confusion exists between Genotypic Sex Determination and Sex Chromosomes, which are not the same thing. All sex chromosome systems are a form of genotypic sex determination but not all GSD systems are sex chromosome systems. Sex chromosomes are linkage groups associated with sex (males, females) but not all consistent differences in chromosomes or genes between the sexes define sex chromosomes (however,  such differences define a GSD system). For instance Haplodiploidy is a GSD system which does not involve sex chromosomes, where sex is determined by the ploidy level such that diploid genotypes induce female development and haploid genotypes induce male development but all chromosomes are autosomes. PGE systems involve either sex chromosomes X0 systems (X0, X1X200), true haplodiploid systems, or functionally haplodiploidy (see Herbette & Ross 2023)

Herbette, M. and L. Ross. 2023. Paternal genome elimination: patterns and mechanisms of drive and silencing. Curr. Opin. Genet. Dev. 81.

**Comment by genomicrocosm** [comment]
Suggestion to replace "linkage group" with "chromosome".

**Comment by PetrNguyen** [comment]
Sex chromosomes are chromosomes whose inheritance is associated with sex
